tests/api/validate.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { validateSchedules } from "@dataform/api";
+import { dataform } from "@dataform/protos";
+import { assert, config, expect } from "chai";
+
+describe("@dataform/api/validate", () => {
+  describe("validateSchedules", () => {
+
+    it("test_valid_schedule", () => {
+      const compiledGraph = dataform.CompiledGraph.create({
+        tables: [
+          {
+            name: "action1"
+          },
+          {
+            name: "action2"
+          }
+        ]
+      });
+      const valid_schedule = dataform.schedules.SchedulesJSON.create({
+        defaultNotification: {
+          emails: ["team@dataform.co", "abc@test.com"],
+          success: true,
+          failure: true
+        },
+        schedules: [
+          {
+            name: "name1",
+            cron: "*/2 * * * *",
+            enabled: false,
+            options: {
+              actions: ["action2"]
+            }
+          },
+          {
+            name: "name2",
+            cron: "*/2 * * * *",
+            notification: {
+              emails: ['tada@test.com'],
```

Comment:
nit: our prettifier style should be replacing these `'` with `"`, can you double check that you have it installed / that it runs on save?